ndc-models/src/lib.rs

```diff
@@ -679,13 +682,23 @@ pub struct NestedArray {
 }
 // ANCHOR_END: NestedArray
 
+// ANCHOR: ArrayOfObjects
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+#[schemars(title = "ArrayOfObjects")]
+pub struct ArrayOfObjects {
+    pub query: Query,
+}
+// ANCHOR_END: ArrayOfObjects
+
 // ANCHOR: NestedField
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize, JsonSchema)]
 #[serde(tag = "type", rename_all = "snake_case")]
 #[schemars(title = "NestedField")]
 pub enum NestedField {
     Object(NestedObject),
     Array(NestedArray),
+    ArrayOfObjects(ArrayOfObjects),
```

Comment:
We refer to this feature as "nested collections" everywhere except on the actual types... perhaps we should just change this to `NestedCollection(NestedCollection)`?

Reply:
I agree that `NestedCollection` is a better name